nginx/etc/nginx/templates/nginx.conf

```diff
@@ -0,0 +1,48 @@
+# This is an example Nginx configuration template file.
+# Adjust the values below as required for your application.
+
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    map $status $isNot2xx {
+        ~^2 0;
+        default 1;
+    }
+    
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for" '
+                      '$upstream_addr';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile           on;
+    keepalive_timeout  65;
+
+    {{ $ssl_ready := env "SSL_READY" }}
+
+    {{ if eq $ssl_ready "true" }}
+    include ssl.conf;
+    {{ end }}
+
+    {{ if service "wordpress" }}
```

Comment:
This block inside `{{ if service "wordpress" }}` is the only block that's needed here. Everything else is likely to be repeated for other consumers of the Nginx image.